TSPL.docc/LanguageGuide/Concurrency.md

```diff
@@ -287,37 +296,70 @@ move(firstPhoto, from: "Summer Vacation", to: "Road Trip")
 In the example above,
 because the `move(_:from:to:)` function is synchronous,
 you guarantee that it can never contain possible suspension points.
+This encapsulates the code that temporarily makes the data model inconsistent,
+and makes it easier for anyone reading the code
+to recognize that no other code can run
+before data consistency is restored by completing the work.
 In the future,
 if you try to add concurrent code to this function,
 introducing a possible suspension point,
 you'll get compile-time error instead of introducing a bug.
 
-<!--
-  TODO you can also explicitly insert a suspension point
-  by calling ``Task.yield()``
-  https://developer.apple.com/documentation/swift/task/3814840-yield
--->
-
 <!--
   TODO add detail above about how the *compiler* can reason about
   the async/await version better too
   and give you better guarantees and clearer errors
 -->
 
-> Note: The [`Task.sleep(until:tolerance:clock:)`](https://developer.apple.com/documentation/swift/task/sleep(until:tolerance:clock:)) method
-> is useful when writing simple code
-> to learn how concurrency works.
-> This method does nothing,
-> but waits at least the given number of nanoseconds before it returns.
-> Here's a version of the `listPhotos(inGallery:)` function
-> that uses `sleep(until:tolerance:clock:)` to simulate waiting for a network operation:
->
-> ```swift
-> func listPhotos(inGallery name: String) async throws -> [String] {
->     try await Task.sleep(until: .now + .seconds(2), clock: .continuous)
->     return ["IMG001", "IMG99", "IMG0404"]
-> }
-> ```
+The example above showed an approach you can use
+to prevent a piece of code from containing any possible suspension points.
+In contrast,
+you can explicitly insert a suspension point
```

Comment:
Because the previous sentence already spelled it out as "possible suspension point", I was using the shorter phrasing here, following [TSPL's style guide](https://github.com/apple/swift-book/blob/main/Style.md#potential-suspension-point).  Were you calling this out because it's ambiguous?

If you disagree with that style rule, we can also change it.  I see from `git grep` that current usage is about 50/50:

```
% git grep -i 'suspension point' | grep 'possible suspension point' | nl
     1	LanguageGuide/Concurrency.md:to mark the possible suspension point.
     2	LanguageGuide/Concurrency.md:which means every possible suspension point is marked with `await`.
     3	LanguageGuide/Concurrency.md:   there aren't any possible suspension points.
     4	LanguageGuide/Concurrency.md:The possible suspension points in your code marked with `await`
     5	LanguageGuide/Concurrency.md:Code in between possible suspension points runs sequentially,
     6	LanguageGuide/Concurrency.md:so there aren't any possible suspension points in this code.
     7	LanguageGuide/Concurrency.md:you guarantee that it can never contain possible suspension points.
     8	LanguageGuide/Concurrency.md:introducing a possible suspension point,
     9	LanguageGuide/Concurrency.md:to prevent a piece of code from containing any possible suspension points.
    10	LanguageGuide/Concurrency.md:Explicitly inserting a possible suspension point with `Task.yield()`
    11	LanguageGuide/Concurrency.md:writing `await` indicates a possible suspension point.
    12	LanguageGuide/Concurrency.md:- In both cases, you mark the possible suspension point with `await`
    13	LanguageGuide/Concurrency.md:accessing `logger.max` is a possible suspension point.

% git grep -i 'suspension point' | grep -v 'possible suspension point' | nl
     1	Style.md:## potential suspension point
     2	Style.md:you can shorten it to just “suspension point”.
     3	Style.md:## suspension point
     4	Style.md:See entry for *potential suspension point*.
     5	LanguageGuide/Concurrency.md:   That code also runs until the next suspension point, marked by `await`,
     6	LanguageGuide/Concurrency.md:you can explicitly insert a suspension point
     7	LanguageGuide/Concurrency.md:it can't contain any potential suspension points.
     8	LanguageGuide/Concurrency.md:Without the suspension point,
     9	LanguageGuide/Concurrency.md:you use `await` to mark the potential suspension point.
    10	LanguageGuide/Concurrency.md:only in places where `await` marks a suspension point.
    11	LanguageGuide/Concurrency.md:Because `update(with:)` doesn't contain any suspension points,
    12	LanguageGuide/Concurrency.md:   because suspension points allow for interleaving.
    13	ReferenceManual/Expressions.md:An expression marked with `await` is called a *potential suspension point*.
    14	ReferenceManual/Expressions.md:This means code between potential suspension points
    15	ReferenceManual/Expressions.md:before the next potential suspension point.
```